setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This whole file is out of scope. We don't want to maintain instructions for a specific system. It must be deleted entirely.

Reply:
Its an example file. Thats all may be just denote it like that.